content/staff-wellbeing/supporting-staff-mental-health.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Supporting staff mental health in schools
+---
+
+# Supporting staff mental health in schools
+
+[Promoting and supporting mental health and wellbeing in schools and colleges](https://www.gov.uk/guidance/mental-health-and-wellbeing-support-in-schools-and-colleges)
+
+Find out what help you can get to develop a whole school or college approach to
+mental health and wellbeing.
+
+<hr class="section-break--thin"></hr>
+
+[Education Support's resources](https://www.educationsupport.org.uk/get-help/help-for-you/helpline/)
+
+To help your organisation with mental health and wellbeing including a free 24
+hour helpline for education staff.
+
+<hr class="section-break--thin"></hr>
+
+[Mentally Healthy Schools](https://www.mentallyhealthyschools.org.uk/)
+
+A directory of mental health resources for schools and further education
+settings in England, Northern Ireland, Scotland and Wales.
+
+<hr class="section-break--thin"></hr>
+
+[Anna Freud's guidance on supporting staff wellbeing in schools](https://www.annafreud.org/resources/schools-and-colleges/supporting-staff-wellbeing-in-schools/)
+
+Guidance on what school staf and senior leaders can do to support their own and
+their colleagues' wellbeing.
+
+<hr class="section-break--thin"></hr>
+
+[Mental Health at Work resources for schools](https://www.mentalhealthatwork.org.uk/?s=resources+for+schools)
+
+<hr class="section-break--thin"></hr>
+
+[Headspace for Educators](https://www.headspace.com/educators)
+
+Mindfulness and meditation sessions for teachers and supporting staff.
+
+<hr class="section-break--thin"></hr>
+
+[Staff wellbeing guide for schools and trusts from the National Employer's Organisation for School Teachers](https://www.local.gov.uk/our-support/workforce-and-hr-support/education-and-young-people)
+
+Guidance and resourcs for schools and trusts to support the wellbeing of their
+staff, especially during the COVID-19 pandemic.
```

Comment:
could we please remove 'especially during the COVID-19 pandemic? I think this is copying content from an old GOVUK page and it no longer feels as relevant